setup.py

```diff
@@ -13,7 +13,7 @@
     author='Vasu Kulkarni',
     author_email='vasu@redhat.com',
     install_requires=[
-        'apache-libcloud',
+        'apache-libcloud==2.6.1',
```

Comment:
Is this causing any issues?

Reply:
Yes, we are hitting invalid osp credentials issue with latest 2.8 version of apache-libcloud version.